qcodes/instrument/parameter.py

```diff
@@ -2038,6 +2071,21 @@ def max_val_age(self) -> Optional[float]:
         """
         return self._max_val_age
 
+    @property
+    def valid(self) -> bool:
+        """
+        Returns True if the cache is expected be be valid.
```

Comment:
"expected" to? can't we just say "if the cache is valid"?
```suggestion
        Returns True if the cache is expected to be valid.
```